pkg/kubelet/kubelet_node_status.go

```diff
@@ -592,7 +592,7 @@ func (kl *Kubelet) setNodeStatusMachineInfo(node *v1.Node) {
 		devicePluginCapacity, devicePluginAllocatable, removedDevicePlugins = kl.containerManager.GetDevicePluginResourceCapacity()
 		if devicePluginCapacity != nil {
 			for k, v := range devicePluginCapacity {
-				glog.V(2).Infof("Update capacity for %s to %d", k, v.Value())
```

Comment:
Probably not everytime, but is it possible we print the log message if capacity actually changes. It did help debug some race condition during node startup in the past.

Reply:
Done